src/Widgets/FormHiddenFieldWidget/FormHiddenFieldWidgetEditingConfig.js

```diff
@@ -19,7 +19,10 @@ Scrivito.provideEditingConfig("FormHiddenFieldWidget", {
   },
   validations: [customFieldNameValidation],
   titleForContent: (widget) =>
-    [widget.get("customFieldName"), truncate(widget.get("hiddenValue"))]
-      .filter((e) => e)
-      .join(": "),
+    truncate(
+      `Hidden: ${[widget.get("customFieldName"), widget.get("hiddenValue")]
```

Comment:
* [ ] The title is not in line with other form fields. E.g. "Form Checkbox", "Form Input Field", "Form Button". Thus "Hidden Form Field" would be more consistent.